File: OPENAPI-REST-API/openapi-client/cpp-tiny/lib/Models/VpsTrafficHistorySectionDataResponse.cpp


#include "VpsTrafficHistorySectionDataResponse.h"

using namespace Tiny;

VpsTrafficHistorySectionDataResponse::VpsTrafficHistorySectionDataResponse()
{
}

VpsTrafficHistorySectionDataResponse::VpsTrafficHistorySectionDataResponse(std::string jsonString)
{
	this->fromJson(jsonString);
}

VpsTrafficHistorySectionDataResponse::~VpsTrafficHistorySectionDataResponse()
{

}

void
VpsTrafficHistorySectionDataResponse::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);


}

bourne::json
VpsTrafficHistorySectionDataResponse::toJson()
{
    bourne::json object = bourne::json::object();


    return object;

}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/vps_traffic_history_section_data_response.dart
part of swagger.api;

class VpsTrafficHistorySectionDataResponse {
  
  VpsTrafficHistorySectionDataResponse();

  @override
  String toString() {
    return 'VpsTrafficHistorySectionDataResponse[]';
  }

  VpsTrafficHistorySectionDataResponse.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
  }

  Map<String, dynamic> toJson() {
    return {
     };
  }

  static List<VpsTrafficHistorySectionDataResponse> listFromJson(List<dynamic> json) {
    return json == null ? new List<VpsTrafficHistorySectionDataResponse>() : json.map((value) => new VpsTrafficHistorySectionDataResponse.fromJson(value)).toList();
  }

  static Map<String, VpsTrafficHistorySectionDataResponse> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, VpsTrafficHistorySectionDataResponse>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new VpsTrafficHistorySectionDataResponse.fromJson(value));
    }
    return map;
  }
}
